common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -181,10 +200,70 @@ def student_view(self, context):
                   'ajax_url': self.system.ajax_url,
                   }
 
-        fragment.add_content(self.system.render_template('seq_module.html', params))
+        fragment.add_content(self.system.render_template("seq_module.html", params))
 
         return fragment
 
+    def _time_limited_student_view(self, context):
+        """
+        Delegated rendering of a student view when in a time
+        limited view. This ultimately calls down into edx_proctoring
+        pip installed djangoapp
+        """
+
+        # None = no overriden view rendering
+        view_html = None
+
+        # Is this sequent part of a timed or proctored exam?
+        proctoring_service = self.runtime.service(self, 'proctoring')
+        credit_service = self.runtime.service(self, 'credit')
+
+        # Is the feature turned on?
```

Comment:
This comment is a bit redundant with the code -- I'd either drop it or make it more useful